test/MathOptInterface/MOI_wrapper.jl

```diff
@@ -115,19 +114,18 @@ end
     MOIT.lin3test(BRIDGED_CERTIFICATE_OPTIMIZER, CONFIG)
     MOIT.lin4test(BRIDGED_CERTIFICATE_OPTIMIZER, CONFIG)
 
-    # TODO(odow): duals for SOC constraints.
-    soc_config = MOIT.TestConfig(duals = false, atol=5e-3)
+    soc_config = MOIT.TestConfig(atol = 1e-3, infeas_certificates = false)
 
-    MOIT.soctest(BRIDGED_OPTIMIZER, soc_config, [
-        "soc3"
-    ])
-
-    MOIT.soc3test(
-        BRIDGED_OPTIMIZER,
-        MOIT.TestConfig(duals = false, infeas_certificates = false, atol = 1e-3)
-    )
+    MOIT.soctest(BRIDGED_CERTIFICATE_OPTIMIZER, soc_config)
 
-    MOIT.rsoctest(BRIDGED_OPTIMIZER, soc_config)
+    #TODO: rotatedsoc2 needs to disable dual test because CPLEX fails to give INFEASIBILITY_CERTIFICATE
```

Comment:
If I read this in a few months, I won't know if this is an expected behavior of CPLEX or a bug in CPLEX (and then was it reported upstream yet ?).
Write something like
```
rotatedsoc2 needs to disable dual test because CPLEX fails to give INFEASIBILITY_CERTIFICATE
TODO determine whether this is an expected behavior and CPLEX does not support certifying infeasibility of quadratic problems and file it upstream if it is not.
See https://github.com/JuliaOpt/CPLEX.jl/pull/295#issuecomment-610456477
```

Reply:
@blegat Changes applied.